client/components/redux-forms/redux-form-textarea/index.jsx

```diff
@@ -0,0 +1,29 @@
+/**
+ * External dependencies
+ */
+import React, { PureComponent, PropTypes } from 'react';
+import { Field } from 'redux-form';
+
+/**
+ * Internal dependencies
+ */
+import FormTextarea from 'components/forms/form-textarea';
+
+// eslint-disable-next-line no-unused-vars
+const RenderTextarea = ( { input, meta, type, ...otherProps } ) => {
+	return (
+		<FormTextarea { ...input } { ...otherProps } />
+	);
+};
+
+class ReduxFormTextarea extends PureComponent {
+	static propTypes = {
+		name: PropTypes.string.isRequired,
+	};
+
+	render() {
+		return <Field component={ RenderTextarea } type="text" { ...this.props } />;
+	}
+}
```

Comment:
If the component only holds `propTypes` apart from a `render` method, might be worth considering turning into a functional component.